packages/ckeditor5-table/tests/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -1946,6 +1946,104 @@ describe( 'TableColumnResizeEditing', () => {
 		} );
 	} );
 
+	describe( 'Hide/show resizers', () => {
+		let model, editor, view, editorElement;
+
+		beforeEach( async () => {
+			editorElement = document.createElement( 'div' );
+			document.body.appendChild( editorElement );
+			editor = await createEditor();
+
+			model = editor.model;
+			view = editor.editing.view;
+		} );
+
+		afterEach( async () => {
+			if ( editorElement ) {
+				editorElement.remove();
+			}
+
+			if ( editor ) {
+				await editor.destroy();
+			}
+		} );
+
+		describe( 'on mousedown/mouseup', () => {
+			it( 'should not hide resizers when resizer is clicked', () => {
+				setModelData( model,
+					'<table columnWidths="100%">' +
+					'<tableRow>' +
+					'<tableCell>' +
+					'<paragraph>' +
+					'[foo]' +
+					'</paragraph>' +
+					'</tableCell>' +
+					'</tableRow>' +
+					'</table>'
+				);
+
+				const resizer = view.getDomRoot().querySelector( '.table-column-resizer' );
+
+				resizer.dispatchEvent( new MouseEvent( 'mousedown', { bubbles: true } ) );
+
+				const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );
+
+				expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
+			} );
+
+			it( 'should hide resizers when table cell is clicked and show resizers on mouseup', () => {
```

Comment:
There is one missing chain in our tests. The logic looks like this:

`mousedown` / `mouseup` -> `setResizersEnablement()` -> add/remove class.

We should ensure all the links are tested. The second one (between function and class) is already covered in the `utils` file. The test that we have here skips the first link - it jumps from the start to the end directly. Instead we should check, if the mousedown and mousup events trigger the `setResizersEnablement()` function (and don't trigger them when they shouldn't). The integration test like the one here can also be useful to ensure that all the links are connected into a reasonable chain, but it is not always mandatory.